pennylane/qchem/convert.py

```diff
@@ -1164,25 +1157,15 @@ def _shci_state(wavefunction, tol=1e-15):
 
     **Example**
 
-    >>> from pyscf import gto, scf, mcscf
-    >>> from pyscf.shciscf import shci
     >>> import numpy as np
-    >>> mol = gto.M(atom=[['Li', (0, 0, 0)], ['Li', (0,0,0.71)]], basis='sto6g', symmetry="d2h")
-    >>> myhf = scf.RHF(mol).run()
-    >>> ncas, nelecas_a, nelecas_b = mol.nao, mol.nelectron // 2, mol.nelectron // 2
-    >>> myshci = mcscf.CASCI(myhf, ncas, (nelecas_a, nelecas_b))
-    >>> output_file = f"shci_output.out"
-    >>> myshci.fcisolver = shci.SHCI(myhf.mol)
-    >>> myshci.fcisolver.outputFile = output_file
-    >>> e_tot, e_ci, ss, mo_coeff, mo_energies = myshci.kernel(verbose=5)
-    >>> (dets, coeffs) = [post-process shci_output.out to get tuple of
-        dets (list of strs) and coeffs (list of floats)]
-    >>> wf_shci = _shci_state((dets, coeffs), tol=1e-1)
+    >>> wavefunction = ( ['20', '02'], np.array([-0.9943019036, 0.1066007711]))
```

Comment:
Have we tested this for larger determinant representation that `('20', '02')`?

Reply:
Definitely -- we have a Li2 test in `tests` for SHCI.